modules/local/check_torch_model.nf

```diff
@@ -7,18 +7,21 @@ process CHECK_TORCH_MODEL {
     
     input:
     tuple path(original_csv), path(model),  path(experiment_config), path(ray_tune_config)
+    each path(initial_weights)
 
     output:
     stdout emit: standardout
 
     script:
     def args = task.ext.args ?: ''
+    def args_initial_weights = initial_weights ? "--initial_weights $initial_weights" : ""
```

Comment:
this line is not necessary any more if the suggestion made above on line 23 of launch_check_model is applied. try and see but i'm pretty sure this line can b eremoved and "--initial_weights $initial_weights" can be directly put below.